geode-core/src/main/java/org/apache/geode/cache/client/internal/LiveServerPinger.java

```diff
@@ -41,26 +43,49 @@ public class LiveServerPinger extends EndpointListenerAdapter {
   protected final InternalPool pool;
   protected final long pingIntervalNanos;
 
+  /**
+   * Initial delay offset time between LiveServerPinger tasks. Time in milliseconds.
+   */
+  public static final int INITIAL_DELAY_MULTIPLYER_IN_MILLISECONDS =
+      Integer.getInteger(GeodeGlossary.GEMFIRE_PREFIX
+          + "LiveServerPinger.INITIAL_DELAY_MULTIPLYER_IN_MILLISECONDS", 0);
+
+  private final AtomicInteger offsetIndex = new AtomicInteger(0);
+
+
   public LiveServerPinger(InternalPool pool) {
     this.pool = pool;
-    pingIntervalNanos = ((pool.getPingInterval() + 1) / 2) * NANOS_PER_MS;
+    pingIntervalNanos = TimeUnit.MILLISECONDS.toNanos((pool.getPingInterval() + 1) / 2);
   }
 
   @Override
   public void endpointCrashed(Endpoint endpoint) {
+    offsetIndex.set(0); // Reset counter
```

Comment:
If this statement really needs this comment then I suggest making it a method. I also suggest changing the member name a little.

```java
private void resetInitialDelay() {
  intitialDelay.set(0);
}
```